logs/README.md

```diff
@@ -0,0 +1,122 @@
+# Logs
+
+## Log object
+
+```json
+{
+  "type": "string",
+  "meta": "object",
+  "body": "object",
+  "timestamp": "object"
+}
+```
+
+## **Requests**
+
+|               Route               |                      Description                       |
+| :-------------------------------: | :----------------------------------------------------: |
+| [GET /logs/cache](#get-logscache) | Returns the cache logs and its count of last 24 hours  |
+| [GET /logs/:type](#get-logstype)  | Returns all logs according to the provided type of log |
+
+---
+
+## **GET /logs/cache**
+
+Returns the cache logs and its count of last 24 hours.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Body**  
+  None
+- **Headers**  
+  None
+- **Cookie**  
+  rds-session: `<JWT>`
+
+- **Success Response:**
+
+  - **Code:** 200
+
+    - **Content:**
+
+    **Note: **LOGS_COUNT is a number
+
+    ```json
+    {
+      "message": "Cache Logs returned successfully!",
+      "count": "<LOGS_COUNT>",
+      "logs": ["<LOG_OBJECT>", "<LOG_OBJECT>...."]
+    }
+    ```
+
+- **Error Responses:**
+  - **Code:** 401
+    - **Content:**
+    ```json
+    {
+      "statusCode": 401,
+      "error": "Unauthorized",
+      "message": "Unauthenticated User"
+    }
+    ```
+  - **Code:** 500
+    - **Content:**
+    ```json
+    {
+      "statusCode": 500,
+      "error": "Internal Server Error",
+      "message": "Something went wrong. Please contact admin"
+    }
+    ```
+
+## **GET /logs/:type**
+
+**Note :**
+
+- This API can only be accessed by super user
+
+Returns all logs according to the provided queries and params.
+
+- **Params**  
```

Comment:
What is the difference between params and query?

Reply:
Ah, I see here (in the `/logs/:type` API), that this can also be achieved by the queries `?type=<TYPE_OF_LOG>`, there is no need for this route maybe now as we are only filtering out the data with `type` and this route can rename to `/logs` only.
But not sure why @pallabez made it that way
PR for the same
https://github.com/Real-Dev-Squad/website-backend/pull/526

-------------------------------------------------------------------------------------------------------------------------------------
**Below are my thoughts for arriving at the above conclusion**

I do not know the exact definition but can explain it with examples.
Params and queries both can be attached to routes and are used for searching out data, but here params target the specific property in the data present 
**For example:**
Here, we have attached `: type` to the route, so in the log object, we can see there is a `type` property present so we filter out data according to the type here. Also, a general example can be, 
Suppose we have 10 users which have an API, `/users` and each user has their own unique id, now we want complete data of a specific user, so here we can use param as id `/users/:id` and extract out that specific user
```
{
  "type": "string",
  "meta": "object",
  "body": "object",
  "timestamp": "object"
}
```
whereas queries kind of does the same thing........

Reply:
Changed the heading to `Path Params` to get more clear picture